### ISSUE TITLE
SOLR Support

### ISSUE THREAD

**Issue by karlnicholas** [SFMOCI/openlaw#6]
Is there solr search engine support here? I noticed they are using in the openstates development thread.

**Comment by jasonlally** [comment]
@cmbirk Do you have any insight on SOLR support on San Francisco Decoded?

**Comment by cmbirk** [comment]
It's coming soon.  We were running into some issues with scaling SOLR to all of our sites, but it sounds like the issue will be resolved very soon.

**Comment by karlnicholas** [comment]
They seem to be heavy into finishing up version 0.8. Are you all thinking of upgrading to that version with Solr? Thanks, Karl.

**Comment by cmbirk** [comment]
Yep.  We're upgrading to the newest version which comes with a redesign as well as SOLR support.

**Comment by karlnicholas** [comment]
Awesome, I'll be psyched to see it up and running.
